### ISSUE TITLE
Regrowth time addition to Time Tracker plugin

### ISSUE THREAD

**Issue by nelsondcds** [runelite/runelite#12522]
![Time Tracking](https://user-images.githubusercontent.com/69007111/93721102-a9f19900-fb5b-11ea-9d01-b5401374db4d.jpg)

Currently, the time tracker will tell you the approximated time of when something you've planted will be fully grown. Once the plant has fully grown and been harvested, if the patch is not automatically cleared - aka it is not an herb or allotment patch - the time tracker will begin telling you how much the patch has regrown, such as the coconut tree above. As can be seen in the image, the regrowth is already tracked, but no approximate time is given for when it will be done regrowing. 

Could a toggle feature be added to the time tracker to add regrowth time approximations?

**Comment by Alexsuperfly** [comment]
so to be succinct, you would like a timer to show when the fruit is completely regrown and can be fully harvested again

**Comment by nelsondcds** [comment]
Yes, that is correct. I tend to be awful at being succinct so thank you for the assist there @Alexsuperfly

**Comment by nelsondcds** [comment]
![Time Tracking pt2](https://user-images.githubusercontent.com/69007111/93813274-d4f1f080-fc20-11ea-8e56-70f46042c527.jpg)

I've now found that this is actually already done on the overview window, which makes it extra strange to me that the timer doesn't show on the specific patches.
